app/js/enhancedDate.js

```diff
@@ -1,5 +1,94 @@
 /* 
  * add your module definition for enhancedDate here
  * assign the module to a global variable
- * /
+ */
+
+var enhancedDate = (function enhancedDate() {
+
+    'use strict';
+ 		 
+	var date = null;
+	var isDateSet = false;
+	var days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 
+			'Friday', 'Saturday'];
+	var months = ['Januray', 'February', 'March', 'April', 'May', 'June',
+			'July', 'August', 'September', 'November', 'December'];
+
+    var publicAPI = {
+
+    	setDate: function setDate(myDate) {
+
+    		if (myDate == "Invalid Date") {
```

Comment:
Always use === over ==. More importantly, your check for "Invalid Date" seems like test code. There's two ways to approach how to handle input here, and the assignment specification didn't prescribe one way or the other, so either would work: 1. anything that isn't a Date object or a millisecond value just ignore and create a new Date object, passing in nothing to the constructor. 2. if a value was passed in, anything that isn't a Date object or a millisecond value is invalid. In that case, you wouldn't check for "Invalid Date", you would just check that it's not a Date object, a millisecond value, or nothing.